Arrays

DZemitis opened the pull request.

DaveBarada committed:
Exercise1 done

DaveBarada committed:
Exercise2 done

DaveBarada committed:
Exercise3 done

DaveBarada committed:
Exercise4 done

DaveBarada committed:
Exercise 5 fix

DaveBarada committed:
Exercise 6 done

DaveBarada committed:
half done with TicTacToe

DaveBarada committed:
changes to 2 and TicTacToe

DaveBarada committed:
fix for Exercise4

DaveBarada committed:
stuff with TicTacToe

DaveBarada committed:
tic tac toe done

DaveBarada committed:
small fix

DaveBarada committed:
Exercise 8 done

ausmoons commented on csharp-basics/exercises/Arrays/Arrays/Exercise 8/Program.cs:
```diff
@@ -0,0 +1,88 @@
+                }
+                else
+                {
+                    if (missesCounter == 0)
```
Else ir lieks. Vari atsevišķu if likt

ausmoons commented on csharp-basics/exercises/Arrays/Arrays/TicTacToe/Program.cs:
```diff
@@ -6,31 +6,150 @@ class Program
-            DisplayBoard();
+            bool playAgain = true;
+
+            while (playAgain == true)
```
Vai tad šeit neder arī while (playAgain)?

ausmoons commented on csharp-basics/exercises/Arrays/Arrays/TicTacToe/Program.cs:
```diff
@@ -6,31 +6,150 @@ class Program
+                else if (player == board[2, 0] && player == board[2, 1] && player == board[2, 2])
+                {
+                    Console.Clear();
+                    Console.WriteLine($"Congratulations, {player} has won the game!");
```
Tev vairākas reizes atkārtojas viens un tas pats. Apvieno.

ausmoons commented on csharp-basics/exercises/Arrays/Arrays/TicTacToe/Program.cs:
```diff
@@ -6,31 +6,150 @@ class Program
+            }
+        }
+
+        private static void addData(out int row, out int col)
```
Metodes ar lielo burtu

DaveBarada committed:
Exercise_8 changed else to separate if.

TicTacToe
Changed method naming to capital letter of first index.
Redone if statement so it does not repeat else if.
Changed PlayAgain in while loop condition, as its true in default.
